presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -243,4 +261,28 @@ private MaterializedResult getExpectedOrdersTableDescription(boolean dateSupport
                     .build();
         }
     }
+
+    @Test
+    public void testCreateSchema()
+            throws SQLException
+    {
+        skipTestUnless(canCreateSchema());
+        assertUpdate("CREATE SCHEMA test_create_schema");
```

Comment:
test could be run concurrently and single connector can have multiple implementation of this abstract class. Using hardcoded name will make this test to be flaky. Please use `io.prestosql.testing.sql.TestTable#randomTableSuffix` to make this schema name unique. Some for test below.

Currently you are using same schema name in this test and in `testDropSchema` so they are flaky even if there is a single implementation of this class and concurrent tests execution.